src/views/Records/Record.vue

```diff
@@ -16,16 +16,25 @@
       </v-row>
 
       <v-row
-        v-if="user().isLoggedIn && !error && canEdit"
+        v-if="user().isLoggedIn && !error"
         class="pr-3"
       >
         <v-spacer />
         <v-btn
+          v-if="canEdit"
           class="success"
           @click="goToEdit()"
         >
           EDIT
         </v-btn>
+        <v-btn
```

Comment:
Do we agree that if a user can both edit and claim, then two buttons would ?
You may want to use a v-else-if="mycondition" instead.
Not a request, just a thing to consider.

Reply:
This is one I'm not completely sure about. Technically, curators don't need to claim anything, so I was thinking about allowing them to assign maintainer roles to themselves directly in an admin interface to be developed later. They may indeed request this, though.